plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1470,6 +1470,8 @@ producer_plugin::get_unapplied_transactions( const get_unapplied_transactions_pa
             return "forked";
          case trx_enum_type::aborted:
             return "aborted";
+         case trx_enum_type::incoming_persisted:
+            return "incoming_persisted";
```

Comment:
Thanks. That is what I get for letting CLion generate the switch statement for me. I wonder why it got confused.